components/resend/app/resend.app.ts

```diff
@@ -1,13 +1,43 @@
 import { defineApp } from "@pipedream/types";
+import { axios } from "@pipedream/platform";
+import {
+  HttpRequestParams, RetrieveEmailParams, SendEmailParams, SendEmailResponse,
+} from "../common/types";
 
 export default defineApp({
   type: "app",
   app: "resend",
-  propDefinitions: {},
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _getHeaders() {
+      return {
+        "Authorization": `Bearer ${this.$auth.api_key}`,
+        "Content-Type": "application/json",
+      };
+    },
+    async _httpRequest({
```

Comment:
True, they're not required since they're already returning promises, but I think I prefer the keyword there to easily distinguish them as promise-returning functions.